### DISCUSSION TITLE
How to access region locked videos?

### DISCUSSION THREAD

**Discussion by MarlusArthur** [libre-tube/LibreTube#3194] [General]
I live in Brazil and I can't access some Brazilian YouTube videos due to region blocking. I believe that these region-locked videos do not work in Libretube because no instance supports Brazil. if I use the instance "kavin.rocks (Official)", for example, YouTube thinks I'm accessing from the US and due to the regional block the video doesn't work.
I leave a link to a video with this restriction that works perfectly on YouTube/NewPipe but Libretube cannot play it. is there any way around this? 
an instance that works very well for South America is "rivo.lol" but I suffer from the same region blocking problems.

https://youtu.be/UB7Qsy-w_Fc

**Comment by FireMasterK** [comment]
This is because that video is only available in Brazil, but there are no Brazilian instances currently.
